### DISCUSSION TITLE
Having Trouble Installing with Ubuntu (aarch64)

### DISCUSSION THREAD

**Discussion by dennisant** [empriselab/RCareWorld#78] [Help - Setting up / Configuring] [answered]
Hello!

Currently I am following the installation guidelines. I am able to successfully get through most of the instructions up until I'm asked to run the test_install_linux.py. Not quite sure what's going on, but I believe it may be an architecture issue. In the Ubuntu video you have on Youtube, which architecture did you use for Ubuntu? Does this support aarch64? If not, is there a way that it could?

**Comment by YoruCathy** [comment, marked as answer]
I don't think it works with aarch64. For Mac with M chips, we did this trick to pretend to be x86 in the terminal, but it doesn't really work out.

**Reply by dennisant** [reply]
Understood. But I'm using Ubuntu through a virtual machine, which I should be able to avoid using the x86 trick in the terminal. Do you know what version and architecture you guys used for Ubuntu in the youtube video? Also, do you have any suggestions if my only option is to use an aarch64 architecture?

**Reply by YoruCathy** [reply]
We are using x86. See if you can get access to some cloud computing resources (AWS, for instance). We haven't tried RCareWorld there but if they provide x86 option you might want to try that. I'd highly recommend teaming up with other people to get access to the compute resources though. You can even use the discussion forum to seek teammates :)

**Reply by dennisant** [reply]
Ah, I see. Actually, I just tried using google colab and it works for the most part. But it seems like I'm having some envs issues. Any advice on how to resolve this issue I'm getting (in the screenshot attached)? I believe it's coming from inside test_install_linux.py, when the code tries to execute `env = RCareWorld(executable_file="./Build/TestInstall/Linux/loadObject.x86_64", assets=["Cube"] `
<img width="1315" alt="Screenshot 2024-07-23 at 11 58 36 AM" src="https://github.com/user-attachments/assets/3fcb73ab-8b5a-42b9-8848-e4c88e568be1">
